src/main/java/io/stargate/sgv2/jsonapi/api/model/command/table/definition/TableDefinitionDesc.java

```diff
@@ -14,8 +13,6 @@
  *
  * <p>Has the columns and the primary key definition, not the name of the table.
  */
-@JsonPropertyOrder({"name", "definition"})
```

Comment:
Not needed as we are not serializing these definitions (but if we did, had wrong column names :) ).